src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,139 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Entity
+public class Book {
```

Comment:
use getter/setter lombok annotetion

Reply:
`@Data` is not recommended for use with entities, I asked getter/setter annotations...